src/pages/CreateAnalysisPage/RightArrowButton.tsx

```diff
@@ -3,7 +3,7 @@ import { ArrowRightIcon } from '@patternfly/react-icons';
 import React from "react";
 
 interface RightArrowButtonProps { 
-  click: () => void; 
+  click?: () => void; 
```

Comment:
Don't believe we need this to be optional anymore since all instances of `RightArrowButtonProps` pass a `click` prop.
```suggestion
  click: () => void; 
```